### PR TITLE
Cookbook: The section "Adding a REST service" is updated

### LINKED ISSUE #3506: Cookbook: The section "Adding a REST service" is outdated
The part "Adding a REST service" describes how it worked (to add a REST service) before a pretty important change and is no longer valid. Needs to be rewritten.

### PR DESCRIPTION
**Pull request by Pradyumn-NEC**
Fixes #3506 
Cookbook: The section "Adding a REST service" is updated

### CODE REVIEW

File: doc/manuals/devel/cookbook.md [hunk 1]
```diff
@@ -118,62 +118,11 @@ typedef struct RestService
...
-```
-
-`UpdateContextRequest` has a bunch of `fill()` methods (seven `fill()` methods as of March 2017) and if there is no fill-method suited for your demands in `putMetadata()`, then another fill-method must be implemented for `UpdateContextRequest`.
-
-It is easy enough, just copy from an older, similar, fill-method.
-
-Now just add `putMetadata.cpp` to the CMake file `src/lib/serviceRoutinesV2/CMakeLists.txt` and compile the broker. To test that `putMetadata()` works correctly, a new functional test case should be implemented. [The following recipe](#adding-a-functional-test-case) explains how to do that.
-
-To capture "POST/PATCH/XXX /v2/entities/*/attrs/*/metadata/*" and respond with a `405 Method Not Allowed`, please have a look at [the recipe about bad method](#catching-a-405-method-not-allowed).
-
```
Comment (fgalan) [inline doc/manuals/devel/cookbook.md:176]: This block of code should be reviewed and update if something on it is wrong, but not fully removed. If you do so, you are removing a valuable piece of knowledge from the documentation about how new services have to be added.

Related with my comment at https://github.com/telefonicaid/fiware-orion/issues/3506#issuecomment-1854247733

File: doc/manuals/devel/cookbook.md [hunk 2]
```diff
@@ -118,12 +118,12 @@ typedef struct RestService
...
   RequestType   request;          // The type of the request  
   int           components;       // Number of components in the URL path  
-  std::string   compV[10];        // Vector of URL path components. E.g. { "v2", "entities" }  
-  std::string   payloadWord;      // No longer used, should be removed ... ?  
+  std::string   compV[10];        // Vector of URL path components. E.g. { "v2", "entities" }   
   RestTreat     treat;            // service function pointer  
 } RestService;
 ```
 
+
```
Comment (fgalan) [inline doc/manuals/devel/cookbook.md:126]:
```suggestion
```

Remove blank line, please.

**Review by fgalan — APPROVED** [review]
LGTM. Thanks!

**Comment by fgalan** [comment]
@fisuda this PR does some changes to the English documentation. Could you sync the Japanese translation, please? Thanks!

**Comment by fisuda** [comment]
I sent the PR #4453. Thanks

**Pull request merged**